### ISSUE TITLE
Langchain v0.3 breaks Lunary integration

### ISSUE THREAD

**Issue by whimo** [lunary-ai/lunary-py#53]
Hi!
Langchain v0.3 was released last week: https://blog.langchain.dev/announcing-langchain-v0-3/

Lunary integration doesn't work with it, gives multiple errors like this one:
```
ERROR:lunary:An error occurred in `on_chain_start`: 'NoneType' object has no attribute 'get'
Traceback (most recent call last):
  File "/Users/whimo/motleycrew/.venv/lib/python3.12/site-packages/lunary/__init__.py", line 1161, in on_chain_start
    if len(serialized.get("id", [])) > 3
           ^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'get'
```

**Comment by whimo** [comment]
This seems to be a problem on Langchain's side, I've filed an issue there as well: https://github.com/langchain-ai/langchain/issues/26773

**Comment by whimo** [comment]
Should be fixed in the next langchain_core release, but I've created a little PR to take the chain name from kwargs directly, if present: https://github.com/lunary-ai/lunary-py/pull/54

**Comment by hughcrt** [comment]
Hi @whimo, thanks for the contribution. You PR has been merged and a lunary-py 1.1.5 has been published 🙂
